tests/integration/integration_steps/service_steps.py

```diff
@@ -2,6 +2,7 @@
 from cdf_fabric_replicator.extractor import CdfFabricExtractor
 from pandas import DataFrame
 from cdf_fabric_replicator.data_modeling import DataModelingReplicator
+from cdf_fabric_replicator.event import EventsReplicator
```

Comment:
Seems like EventsReplicator is not being referenced here and we may not need this import with this change

Reply:
Good catch, removing